test/libsolidity/syntaxTests/dataLocations/libraryExternalFunction/function_argument_location_specifier_test_external_calldata.sol

```diff
@@ -0,0 +1,4 @@
+library test {
+    function f(bytes calldata) external;
+}
+// ----
```

Comment:
Why do these files have this comment line? Did `isoltest` do that?

Reply:
I was lazy to write them from scratch, wrote the failing one, copied it over and ran `isoltest`. It just doesn't remove it when there are no warnings/errors.